selfdrive/car/volkswagen/values.py

```diff
@@ -62,6 +68,43 @@ class CAR:
   }],
 }
 
+FW_VERSIONS = {
+  CAR.AUDI_A3: {
+    (Ecu.engine, 0x7e0, None): [
+      b'\xf1\x875G0906259L \xf1\x890002',  # 2017 Audi A3 Prestige (CNTC)
```

Comment:
remove all the comments for specific FW versions

Reply:
Done